src/lib389/lib389/cli_conf/backend.py

```diff
@@ -854,49 +854,49 @@ def create_parser(subparsers):
     index_subcommands = index_parser.add_subparsers(help="action")
 
     # Create index
-    add_index_parser = index_subcommands.add_parser('add', help='Set configuration settings for a single backend')
+    add_index_parser = index_subcommands.add_parser('add', help='Add an index')
     add_index_parser.set_defaults(func=backend_add_index)
-    add_index_parser.add_argument('--index-type', required=True, action='append', help='An indexing type: eq, sub, pres, or approximate')
-    add_index_parser.add_argument('--matching-rule', action='append', help='Matching rule for the index')
-    add_index_parser.add_argument('--reindex', action='store_true', help='After adding new index, reindex the database')
-    add_index_parser.add_argument('--attr', required=True, help='The index attribute\'s name')
-    add_index_parser.add_argument('be_name', help='The backend name or suffix to delete')
+    add_index_parser.add_argument('--index-type', required=True, action='append', help='Sets the indexing type (eq, sub, pres, or approximate)')
```

Comment:
There was an old error, "approximate" should be "approx"

Reply:
Fixed